components/Sidebar/index.jsx

```diff
@@ -0,0 +1,81 @@
+import styles from './sidebar.module.css';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import sidebarMenuOptions from 'constants/sidebarMenuOptions';
+import { useState } from 'react';
```

Comment:
this should be the 1st import.First the standard libraries should be imported before importing from other files in the project.